pkg/engine/datasource/graphql_datasource/graphql_datasource.go

```diff
@@ -1262,10 +1264,11 @@ type GraphQLSubscriptionClient interface {
 }
 
 type GraphQLSubscriptionOptions struct {
-	URL    string      `json:"url"`
-	Body   GraphQLBody `json:"body"`
-	Header http.Header `json:"header"`
-	UseSSE bool        `json:"use_sse"`
+	URL       string      `json:"url"`
+	Body      GraphQLBody `json:"body"`
```

Comment:
In conjunction with renaming `SSEMethod` to `SSEMethodPost bool`, I'd change `Body` to `PostBody`.

Reply:
I believe it is not required, cause this body is a graphql payload with query, variables etc.